testsuite/tests/conftest.py

```diff
@@ -235,12 +235,20 @@ def backend(request, openshift, blame, label):
 
 
 @pytest.fixture(scope="module")
-def envoy(request, kuadrant, authorino, openshift, blame, backend, module_label, testconfig) -> Proxy:
+def envoy_template():
```

Comment:
I would not do this in root conftest, we are trying to avoid using envoy modification as much as possible, I would override envoy fixture in wristband conftest instead.